data/src/main/java/com/android254/data/network/models/responses/OrganizersPagedResponse.kt

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2022 DroidconKE
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.android254.data.network.models.responses
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class OrganizersPagedResponse(
+    @SerialName("data") var data: List<OrganizerData>,
+    @SerialName("meta") var meta: Meta?
+)
+
+@Serializable
+data class OrganizerData(
+    @SerialName("id") var id: Int?,
+    @SerialName("name") var name: String?,
+    @SerialName("email") var email: String?,
+    @SerialName("description") var description: String?,
+    @SerialName("facebook") var facebook: String?,
+    @SerialName("twitter") var twitter: String?,
+    @SerialName("instagram") var instagram: String?,
+    @SerialName("logo") var logo: String?,
+    @SerialName("slug") var slug: String?,
+    @SerialName("status") var status: String?,
+    @SerialName("created_at") var createdAt: String?,
+    @SerialName("creater") var creator: CreatorData?,
+    @SerialName("upcoming_events_count") var upcomingEventsCount: Int?,
+    @SerialName("total_events_count") var totalEventsCount: Int?
+)
+
+@Serializable
+data class CreatorData(
```

Comment:
`CreatorData` -> `CreatorResponse` ideally we need to suffix every api response with Response